odl/util/utility.py

```diff
@@ -417,6 +417,39 @@ def normalized_index_expression(indices, shape, int_to_slice=False):
     return tuple(indices)
 
 
+def snr(signal, noise, impl):
+    """Compute the signal-to-noise ratio.
+
+    Parameters
+    ----------
+    signal : `array-like`
+        Noiseless data.
+    noise : `array-like`
+        Noise.
+    impl : {'general', 'dB'}
+        Implementation method.
+        'general' means SNR = s_power / n_power,
```

Comment:
s_power and n_power have not been defined so far, you need to say what they mean.